index.html

```diff
@@ -4,8 +4,82 @@
+     <h3>Jordan Devos</h3>
+     <h4>Talk: Designing for Readability – A Guide to Web Typography<br>
+       Time：10 AM - 12 PM, February 1st, 2021
+     </h4>
+     <h6>
+       About the speaker: Jordan is an expert designer whose focus is on visual communications and whose work integrates strategy, service design, and UX/UI. She has worked in a range of companies, from corporate in-house to boutique design studios. A self-starter who wears many hats, Jordan is in her element leading a project or managing a team.
+     </h6>
+     <p>Description: Traditionally, graphic designers have always studied the fundamentals of typography in terms of print design. But as the role of the designer now spans across all types of digital mediums, typography is a bigger beast to tame. In this talk, Devos will introduce some web typography tips about the web typeface and formatting which leads to the best reading experience.</p>
+      </li>
+    
+      <li class="talk2"> 
+     <img src="https://cdn.glitch.com/0e85a349-5917-49b5-9721-e8ec891917e8%2FPhoto%20of%20Suzanne%20Scacca.png?v=1607880365444"
+        alt="Photo of Suzanne Scacca" />
+      <h3>Suzanne Scacca</h3>
+     <h4>A Reference Guide For Typography In Mobile Web Design <br>
+       Time：13 PM - 15PM, February 1st, 2021
+     </h4>
+     <h6>About the speaker: Suzanne Scacca is a former WordPress implementer, trainer and agency manager who now works as a freelance copywriter. She specializes in crafting marketing, web design, WordPress and millennial-related content for a variety of publications.</h6>
+     <p>Description: In terms of how to handle typography in mobile web design, it appears that simpler and safer works best. In this talk, Scacca will lead to break down the elements designers need to pay attention to in mobile typography and then visit what the research says about how to handle them. </p>
+       </li>
+       
+       <li class="talk3">
+     <img src="https://cdn.glitch.com/0e85a349-5917-49b5-9721-e8ec891917e8%2Fphoto%20ofCameron%20Chapman.png?v=1607880869268"
+        alt="Photo of Cameron Chapman" />
+     <h3>Cameron Chapman</h3>
+     <h4>How to Structure an Effective Typographic Hierarchy <br>
+       Time：16 PM - 18PM, February 1st, 2021
+     </h4>
+     <h6>About the speaker: Cameron Chapman (Orleans, VT) is a freelance designer and writes on the subject of design for Smashing Magazine, Mashable, Webdesigner Depot, Six Revisions, and other prominent design blogs.</h6>
+     <p>Description: Effectively organizing content within a design so that it’s easy to understand and consume is one of a designer’s most important jobs. And since much of the content designers work with is text-based, creating an effective typographic hierarchy is one of the most important things a designer can learn. In this talk, Chapman will introduce the concept of “typographic hierarchy” which is very useful for web designers.</p>
+      </li>
+       
+     </ul>
+   </main>
+   
  </body>
```

Comment:
The closing `body` element is always at the end of your markup, along with the closing `html` element; **all** the text that you want to be visible on a web page **must** be inside the `body` tags.